### DISCUSSION TITLE
Welcome to apicurio.github.io Discussions!

### DISCUSSION THREAD

**Discussion by EricWittmann** [Apicurio/apicurio.github.io#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chiaracielo** [comment]
Hi! I’m Chiara, I’m a UX Designer. I work for 10 years in the same context (a school books publishing house) and, to prove myself on something completely different, I decided to freely contribute to an open-source project such as Apicurio. I hope that my analysis and notes could be helpful.

I did some considerations, that I already shared with @EricWittmann, on your website as is. You can find the analysis and some UX and UI issues [here](https://www.figma.com/proto/SRhyF8X9PXzk2RdYJlRPA4/Apicurio?type=design&node-id=21-48&scaling=scale-down&page-id=0%3A1):


I also did a quick accessibility check [here](https://docs.google.com/spreadsheets/d/1dwm_rL0-ngeKLZctmEN21MTfURPehAENoEH0vX3uKGQ/edit?usp=sharing) you can find the issues I collected.

After that, and since the main UX issues are related to navigation, I tried to wireframe some [navigation proposals](https://www.figma.com/proto/SRhyF8X9PXzk2RdYJlRPA4/Apicurio?type=design&node-id=247-1890&scaling=contain&page-id=234%3A349) that I hope can be a starting point for further discussion.

The whole project is available at this [link](https://www.figma.com/file/SRhyF8X9PXzk2RdYJlRPA4/Apicurio?type=design&node-id=21%3A48&t=2gsMIEiTMGIJAvYs-1).

I'm obviously available for any clarification or for some discussion on what I did.
